nsv13/code/__HELPERS/overmap.dm

```diff
@@ -21,3 +21,35 @@ Helper method to get what ship an observer belongs to for stuff like parallax.
 		last_overmap.mobs_in_ship -= src
 	last_overmap = OM
 	OM.mobs_in_ship += src
+
+/// Finds a turf outside of the overmap
+/atom/proc/GetSafeTurf()
+	if(!SSmapping.level_trait(z, ZTRAIT_OVERMAP))
+		return get_turf(src)
+
+	var/max = world.maxx - TRANSITIONEDGE
+	var/min = TRANSITIONEDGE + 1
+	var/list/possible_transitions = list()
+	for(var/datum/space_level/D as() in SSmapping.z_list)
+		if(D.linkage == CROSSLINKED && !D.traits[ZTRAIT_OVERMAP])
```

Comment:
we don't keep the empty space levels around, so we might not need the crosslinked check